cmd/edgex-secrets-config/README.md

```diff
@@ -0,0 +1,181 @@
+
+    * **--incert** _/path/to/certchain.crt_ (required)
+
+      Path to TLS leaf certificate (PEM-encoded x.509) (the file extension is arbitrary).
+      If intermediate certificates are required to chain to a certificate authority,
+      these should also be included.
+      The root certificate authority should not be included.
+
+    * **--inkey** _/path/to/private.key_ (required)
+
+      Path to TLS private key (PEM-encoded).
+
+  * **adduser**
+
+    Create a API gateway user using specified token type. Requires additional arguments:
+
+    * **--token-type** jwt | oauth2 (required)
+    
+      Create user using either the JWT or OAuth2 authentication plugin.
+      This value must match the configured authentication plugin.
+
+    * **--user** _username_ (required)
+
+      Username of the user to add.
+
+    * **--group** _group_ (optional)
+
+      Group to which the user belongs, defaults to &quot;admin&quot;.
+      This should be the group associated with the route ACL.
+      This value can be changed by editing the `[KongACL]` section of configuration.toml.
+      (Note that edgex-secrets-config shares the same configuration as securiry-proxy-setup
+      as they both configure the EdgeX API gateway.)
+
+
+    The following options are used when token-type == "jwt":
+
+    * **--algorithm** RS256 | ES256 (required for JWT method)
+
+      Algorithm used for signing the JWT.
+      (See [RFC 7518](https://tools.ietf.org/html/rfc7518#section-3.1) for a list of signing algorithms.)
+
+    * **--public\_key** _/path/to/public\_key.pem_ (required for JWT tokens)
+
+      Public key (in PEM format) used to validate the JWT.
+      (Not an x.509 certificate.)
+      This key is assumed to have been pre-created using some external mechanism such as a TPM, HSM, openssl, or other method.
+
+    * **--id** _key_ (optional)
+
+      Optional user-specified &quot;key&quot; used for linkage with an incoming JWT via Kong&#39;s config.key\_claim\_name setting (defaults to &quot;iss&quot; field).
+      See
+      [Kong documentation for JWT plugin](https://docs.konghq.com/hub/kong-inc/jwt/#craft-a-jwt-with-publicprivate-keys-rs256-or-es256)
+      for an example of how this parameter is used.
+
+
+    The following options are used when token-type == "oauth2":
+
+    * **--client\_id** (optional)
+
+      Optional manually-specified OAuth2 client_id.  Will be generated if not present.  Equivalent to a username.
+
+    * **--client\_secret** (optional)
+
+      Optional manually-specified OAuth2 client_secret.  Will be generated if not present.  Equivalent to a password.
+
+    * **--redirect\_uris** _url\_for\_browser\_redirection_ (required for oauth2 tokens)
+
+      OAuth2 redirect URL for browser-based users.  Not currently used by EdgeX.
+
+
+  * **deluser**
+
+    Delete a API gateway user. Requires additional arguments:
+
+    * **--token-type** jwt | oauth2 (required)
```

Comment:
You are correct. This is now dead code and dead documenation.  Removed.  Kong doesn't care because there is a cascading delete trigger that clears out the credentials (either jwt or oauth2) when the user is deleted.